tests/unit/test_property.py

```diff
@@ -46,3 +48,22 @@ def test_json_with_tenants(self, create_property, create_lease):
         tenant = lease.tenant
 
         assert property.json(include_tenants=True)["tenants"] == [tenant.json()]
+
+    def test_only_active_tenants_included(self, create_lease):
+        active_lease = create_lease()
+
+        old_lease_end = Time._yesterday()
+        old_lease_start = old_lease_end - relativedelta(years=1)
+        expired_lease = create_lease(
+            property=active_lease.property,
+            dateTimeStart=old_lease_start,
+            dateTimeEnd=old_lease_end,
+        )
+
+        property = PropertyModel.find(expired_lease.propertyID)
+        property_json = property.json(include_tenants=True)
+
+        assert len(property_json["tenants"]) == 1
+        assert len(property_json["leases"]) == 2
+        assert active_lease.tenant.json() in property_json["tenants"]
+        assert expired_lease.tenant.json() not in property_json["tenants"]
```

Comment:
This comment is kinda picky. It seems like we have more assertions than what is needed. 
These assertions are confusing because I have to understand the context before knowing what the assertions mean.
```
        assert len(property_json["tenants"]) == 1
        assert len(property_json["leases"]) == 2
```
I don't think we need an assertion on the leases if we're testing the tenants.

For the assertions I think we could simplify it to
`assert property_json["tenants"]) == [active_lease.tenant.json()]`

I believe this one test confirms what three of your tests are testing for.